[Bug]: Where is the source of media_driver/agnostic/Xe_M/Xe_XPM/codec/kernelisa/Xe_XPM_VC1_OLP.c?

### Which component impacted?

Build

### Is it regression? Good in old configuration?

Yes, it's good in old version

### What happened?

I'm trying to upgrade the free kernel build of the media-driver package in Debian. With the current version, the build requires media_driver/agnostic/Xe_M/Xe_XPM/codec/kernelisa/Xe_XPM_VC1_OLP.c if built with `-DENABLE_KERNELS=ON -DENABLE_NONFREE_KERNELS=OFF`. I am however unable to find the source for this shader. Please confirm the status of this shader with and add the source if it is a free shader or otherwise skip it if the non-free shaders are disabled.

### What's the usage scenario when you are seeing the problem?

Others

### What impacted?

_No response_

### Debug Information

_No response_

### Do you want to contribute a patch to fix the issue?

No.

To reproduce the issue, do this:
```
$ git rm media_driver/agnostic/Xe_M/Xe_XPM/codec/kernelisa/Xe_XPM_VC1_OLP.c
rm 'media_driver/agnostic/Xe_M/Xe_XPM/codec/kernelisa/Xe_XPM_VC1_OLP.c'

$ cd _build/
$ cmake -DENABLE_KERNELS=ON -DENABLE_NONFREE_KERNELS=OFF ..
-- The C compiler identification is GNU 7.3.1
-- The CXX compiler identification is GNU 7.3.1

<.....>

CMake Error at media_driver/media_top_cmake.cmake:195 (add_library):
  Cannot find source file:

    /home/dvrogozh/git/github/media-driver/media_driver/agnostic/Xe_M/Xe_XPM/codec/kernelisa/Xe_XPM_VC1_OLP.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx
Call Stack (most recent call first):
  media_driver/CMakeLists.txt:95 (include)


CMake Error at media_driver/media_top_cmake.cmake:195 (add_library):
  No SOURCES given to target: iHD_drv_video_CODEC
Call Stack (most recent call first):
  media_driver/CMakeLists.txt:95 (include)


-- Build files have been written to: /home/dvrogozh/git/github/media-driver/_build
```

--- a/media_driver/agnostic/Xe_M/Xe_XPM/codec/media_srcs.cmake
+++ b/media_driver/agnostic/Xe_M/Xe_XPM/codec/media_srcs.cmake
@@ -19,4 +19,6 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 
 media_include_subdirectory(hal)
+if(ENABLE_KERNELS AND ENABLE_NONFREE_KERNELS)
 media_include_subdirectory(kernelisa)
+endif()

